tests/integration/minibroker_test.go

```diff
@@ -109,9 +120,10 @@ var _ = Describe("classes", func() {
 				}()
 
 				By("asserting the mariadb client completed successfully")
-				for retry := 0; ; retry++ {
-					if retry == 60 {
-						Fail("maximum retries reached")
+				startTime := time.Now()
```

Comment:
```suggestion
				timeLimit := time.Now().Add(assertTimeout)
```

Since we never use the start time for anything else, we might as well pre-calculate the desired timeout time.